.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,30 @@
+name: 🐞 Bug Report
+title: "[Bug]: "
+labels: ["bug", "needs-triage"]
+assignees: ''
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for letting us know something has happened.
+
+        If this is urgent, please email dotcom.platform@guardian.co.uk, or let us know on our chat channel.
+
+        The benefit of an issue is that we have a written record of what is going on, that we can track and have discussions around.
+        Please still reach out to facilitate discussion or incase we miss it - we're a lovely bunch!
```

```suggestion
        Please still reach out to facilitate discussion or incase we miss it - we're a lovely bunch!
```
        Please still reach out to facilitate discussion or in case we miss it - we're a lovely bunch!